app/models/ability.rb

```diff
@@ -0,0 +1,13 @@
+class Ability
+  include CanCan::Ability
+  def initialize(user)
+    user ||= User.new
```

Comment:
* As stated by the previous CR the error reported to happen while signing up still exists, Kindly fix this issue as i can't successfully sign up and log in. Hint I think This error is often related to email configuration, specifically when trying to send email from your application. 
![image](https://github.com/Simpleshaikh1/blog-app/assets/52497246/0a7e7adc-e939-462c-a981-ec5a96a47de8)

Reply:
but i can signup succesfully on my local